contrib/lkt/language/parser.py

```diff
@@ -647,6 +647,26 @@ def designated_scope():
             default_val=Entity.check_expr_type.type_scope
         )
 
+    @langkit_property(return_type=T.TypeDecl.entity)
+    def array_element_type():
+        # ArrayLiteral elements type is its parent expected type's actuals
+        # type. As class Array takes only one type, actuals.at(0) is the
+        # ArrayLiteral elements type.
```

Comment:
Can you please separate the docstring (user API doc for this property) from the implementation comment? For instance:
```python
"""
If this expression computes an array, return the type of its elements.
Return a null ``TypeDecl`` otherwise.
"""
# As the generic Array class has only one type parameter,
# ``get_actuals.at(0)`` is the element type.
```